backends/prometheus/README.md

```diff
@@ -150,15 +150,15 @@ groups:
           summary: Memory alert for container node '{{ $labels.job }}'
 
       - alert: node_low_root_filesystem_space_20
-        expr: 100 / sum(netdata_disk_space_GB_average{family="/"}) by (job)
-          * sum(netdata_disk_space_GB_average{family="/",dimension=~"avail|cached"}) by (job) < 20
+        expr: 100 / sum(netdata_mount_space_GB_average{family="/"}) by (job)
```

Comment:
I appreciate the update, but the backends documentation will be removed in the next release. :)

Reply:
It is ok. We will most probably not merge this PR. The front-end guys found a better way to do it.